superset-frontend/src/dashboard/components/Header/index.jsx

```diff
@@ -415,7 +415,9 @@ const Header = () => {
       owners: dashboardInfo.owners,
       roles: dashboardInfo.roles,
       slug,
-      tags: dashboardInfo.tags,
+      tags: (dashboardInfo.tags || []).filter(
+        item => item.type === 1 || !item.type,
```

Comment:
do we have some sort of enum/constant anywhere on the frontend that properly labels what TagType == 1 means?

Reply:
found it on the backend -> https://github.com/apache/superset/blob/master/superset/tags/models.py#L61-L77 but couldn't find it on the frontend. Maybe we need a corresponding enum on the frontend if we don't have one already ... hoping we don't hard-code meaningless values related to TagType in other places in the frontend...